frontend/src/types.ts

```diff
@@ -3710,6 +3710,7 @@ export type BatchExportServiceRedshift = {
 // When adding a new option here also add a icon for it to
 // src/scenes/pipeline/icons/
 // and update RenderBatchExportIcon
+export const BatchExportServiceNames = ['S3', 'Snowflake', 'Postgres', 'BigQuery', 'Redshift', 'HTTP']
```

Comment:
JS constants are typically UPPERCASE_SNAKE_CASE
```suggestion
export const BATCH_EXPORT_SERVICE_NAMES = ['S3', 'Snowflake', 'Postgres', 'BigQuery', 'Redshift', 'HTTP']
```